### PR TITLE
Add conda-export package

### PR DESCRIPTION
**Pull request by olegtarasov**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by olegtarasov** [comment]
@conda-forge-admin, please ping team
This package is ready for review. Thank you!

File: recipes/conda-export/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  home: https://github.com/olegtarasov/conda-export
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Platform agnostic conda environment export'
+  description: |
+    An alternative to `conda env export` that helps create portable environment 
+    specifications with minimal number of packages. Resulting specification is 
+    similar to `conda env export --from-history`, but also includes packages 
+    that were installed using `pip`.
```
Comment (ocefpaf) [inline recipes/conda-export/meta.yaml:43]: @olegtarasov I see you are also the author of the code. Do you know about conda-lock? Is the code you use to figure out pip vs conda install something we could use to improve conda-lock?

Comment (jakirkham) [reply recipes/conda-export/meta.yaml:43]: cc @mariusvniekerk

Comment (olegtarasov) [reply recipes/conda-export/meta.yaml:43]: @ocefpaf I didn't know about conda-lock. From what I can gather from the docs, it solves a slightly different problem. The point of conda-export is to create an environment spec file similar to `conda env export --from-history`, adding the packages that were installed separately with pip.

Of course, you are most welcome to use any of my code. But I should warn you that it's pretty naive, probably doesn't handle some tricky edge cases and it uses private `pip` API that can change at any moment. If you find a way to improve it, please ping me so I can incorporate those improvements into conda-export.

Thank you!

Comment (olegtarasov) [reply recipes/conda-export/meta.yaml:43]: @ocefpaf I found a better and less hacky way to get the list of pip packages. See here: https://github.com/olegtarasov/conda-export/blob/b363f508d52bd887f925536fba149d70c2fceb10/conda-export.py#L15

Comment (ocefpaf) [reply recipes/conda-export/meta.yaml:43]: Nice! Simple and elegant. Thanks!

For conda-lock the main issue is that the env is not "realized" yet. In order to support pip packages there we would need to actually create the envs. While it is possible to do without it I'm not sure it is worth it. However, being able to figure out pip packages in an existing env is already a big win for the community. Thanks for conda-export!!

**Pull request merged**